backend/api/v1/auth.py

```
@@ -29,160 +43,140 @@ class UserProfile(BaseModel):
-            school_name=school_name
+        return AuthTokens(
+            access_token=session.access_token,
+            refresh_token=session.refresh_token,
+            token_type="bearer",
+            user=UserPublic(email=user.email, role=role)
         )
-    
+        
     except HTTPException:
         raise
     except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)